modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: PROCEDURE
+
+[id='op-running-as-root-user-namespace_{context}']
+= Running as root in a user namespace
+
+When you run a command in a user namespace, your container runs as root (user id 0) but has user privileges on the host. {product-title} uses CRI-O as its container runtime interface. CRI-O has a workload feature to run a pod in the user namespace.
```

Comment:
Okay, but what is the point of this? That is, how does this help the user namespace? I don't know the value of running root in a user namespace. I think a sentence at the end of this first paragraph that states the value of this would help clarity.